doc/topics/windows/multi-minion.rst

```diff
@@ -0,0 +1,164 @@
+=============================
+Multi-Minion Setup on Windows
+=============================
+
+There may be a scenario where having a minion running in the context of the
+current, logged-in user would be useful. For example, the normal minion running
+under the service account would perform machine-wide, administrative tasks. The
+minion running under the user context could be launched when the user logs in
+and would be able to perform configuration tasks as if it were the user itself.
+
+The steps required to do this are as follows:
+
+1. Create new root_dir
+2. Set root_dir permissions
+3. Create directory structure
+4. Write minion config
+
+We will now go through each of these steps in detail.
```

Comment:
**Suggestion (style, blocking):** Our style guide also wants us to avoid "we" and use the second person instead: https://developers.google.com/style/person

Suggested fix:

The following guide explains these steps in more detail.